src/app/chat-page/chat-page.component.ts

```diff
@@ -179,6 +179,10 @@ export class ChatPageComponent implements OnInit, AfterViewInit {
             show: new_address_text,
             value: {}
           });
+          options.push({
```

Comment:
will show it on first report too, which is not ideal

Reply:
@akariv any other solution than adding a session reports counter? (might be useful later but not sure it worth the effort. WDYT?